tsconfig.json

```diff
@@ -1,9 +1,15 @@
-/* Dev configuration */
 {
-  "extends": "./tsconfig.base.json",
-  "compilerOptions": {
-    "esModuleInterop": true, // Required fto remove warnings or jest test
-  },
-  /* During dev, src are included so they are also modified during refactorings. */
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+    "compilerOptions": {
+        "incremental": true,
+        "target": "es2019",
+        "module": "commonjs",
+        "declaration": true,
+        "sourceMap": true,
+        "composite": true,
+        "strict": true,
+        "moduleResolution": "node",
+        "esModuleInterop": true,
```

Comment:
This is not be able be default: this are only very specific cases why this is needed that should clearly be listed here. Otherwise, this will create less optimal code...

Reply:
Interestingly, this is in the `recommended` base tsconfig:
https://github.com/tsconfig/bases/blob/main/bases/recommended.json

FWIW, the TypeScript team even recommends it in the [release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#support-for-import-d-from-cjs-from-commonjs-modules-with---esmoduleinterop) and imply the recommendation in [the docs](https://www.typescriptlang.org/tsconfig#esModuleInterop):
> Turning on esModuleInterop will fix both of these problems in the code transpiled by TypeScript.

So if anything, this should at least be the default for Node.js

Reply:
I seem to recall `matternode`, which is mostly `js` as opposed to `ts`, required `esModuleInterop: true` as well to link to `matter.js` properly.